01_fib/kaizhu/main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fib(7)
+}
+
+func fib(n int64) {
```

Comment:
`fib` should return an answer. Let the caller do the printing.

Reply:
I disagree with this. From what is written in lab instructions: "Write a function that prints the first n Fibonacci numbers". The fib function should print, not the caller.